backend/src/server.spec.ts

```diff
@@ -5,6 +5,12 @@ import { Body } from './server.interface.js';
 
 describe('Server', () =>
 {
+	afterAll(done =>
+	{
+		mongoose.connection.close();
```

Comment:
does close() properly return a promise? then we could try `close().then(() => done())`